.github/workflows/continuous-integration.yml

```diff
@@ -44,6 +45,10 @@ jobs:
           coverage: "pcov"
           ini-values: "zend.assertions=1"
 
+      - name: "Adjust dependencies"
+        if: "${{ matrix.php-version == '8.1' }}"
+        run: composer config platform.php 8.0.99
```

Comment:
Is it temporary before all dependencies explicitly support 8.1?

Reply:
Yes. Right now Prophecy as a dependency of PHPUnit would block the installation. As soon as that is resolved, we can remove this block.